components/dash-core-components/tests/integration/dropdown/test_dropdown_radioitems_checklist_option_title_prop.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: UTF-8 -*-
+
+import pytest
+from dash.testing import wait
+from dash import Dash, Input, Output, dcc, html
+
+
+@pytest.mark.DCC793
```

Comment:
Non blocking, think we can remove that mark since it refers to old dcc repo issue.